web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx

```diff
@@ -428,19 +427,19 @@ export const MiniAppForm = (props: LinksFormProps) => {
 
       <div className="grid gap-y-5">
         <div className="grid gap-y-3">
-          <Typography variant={TYPOGRAPHY.H7}>Support Email</Typography>
-
+          <Typography variant={TYPOGRAPHY.H7}>Support</Typography>
           <Typography variant={TYPOGRAPHY.R3} className="text-grey-500">
-            Please include a support email where users can reach out to you for
-            help.
+            Please include a support link where users can reach out to you for
+            help. Emails should preceded by mailto:
           </Typography>
         </div>
+        {/* Pending designs change this to a switcher */}
         <Input
-          label="Support Email"
+          label="Support Link"
           disabled={!isEditable || !isEnoughPermissions || !appMode}
-          placeholder="address@example.com"
-          register={register("support_email")}
-          errors={errors.support_email}
+          placeholder="mailto::address@example.com"
```

Comment:
@andy-t-wang Why are you using two colon symbols? Isn't the correct format with the one colon?

Reply:
```suggestion
          placeholder="mailto:address@example.com"
```

Reply:
ah yeah good catch